spec/requests/api/v1/address_validation_spec.rb

```diff
@@ -76,7 +76,7 @@
     expect(json['address']['verifications']['delivery']['success']).to eq(true)
   end
 
-  it 'allows invalid addresses to continue through' do
+  it 'validates invalid address with success == false' do
     VCR.use_cassette('address_validation/invalid') do
```

Comment:
smh this stuff is so rusty that it uses VCR cassettes 🙄

Reply:
😂